packages/wallets/src/features/cfd/screens/ResubmitPOA/ResubmitPOA.tsx

```diff
@@ -0,0 +1,51 @@
+import React, { useState } from 'react';
+import { useSettings, useStatesList } from '@deriv/api';
+import { InlineMessage, WalletDropdown, WalletText, WalletTextField } from '../../../../components/Base';
+import './ResubmitPOA.scss';
+
+const ResubmitPOA: React.FC = () => {
```

Comment:
Shouldnt this be under accounts feature folder?

Reply:
Oh yes 😸